api/config.py

```diff
@@ -84,22 +85,22 @@
 ).get_default_database()
 log.debug(str(db))
 
+es = elasticsearch.Elasticsearch([__config['persistent']['elasticsearch_host']])
```

Comment:
Not really, but all variables -- except `db` -- that are used outside of the config module are periodically updated with the latest DB values. Arguably, `es` is similar to `db` in that a socket connection is established or at least configured. If we really wanted to refresh at runtime, we should at least detect if the config params have actually changed. Come to think of it, it seems like overkill right now. Will discuss with @rentzso on Monday, but inclined to leave as is.